cfep-00.md

```diff
@@ -0,0 +1,31 @@
+
+<table>
+<tr><td> Title </td><td> A short title of the proposal </td>
+<tr><td> Status </td><td> Draft | Proposed | Accepted | Rejected | Deferred | Implemented </td></tr>
+<tr><td> Author(s) </td><td> Full Name &lt;full.name@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Aug 30, 2016</td></tr>
+<tr><td> Updated </td><td> Aug 30, 2016</td></tr>
+<tr><td> Discussion </td><td> link to the issue where the CFEP is being discussed, NA is circulated initially </td></tr>
```

Comment:
This will be a link to the *pull request* for the proposal, not an issue, right?